codbex-electra-ext-econt/api/SalesOrders.js

```diff
@@ -0,0 +1,39 @@
+const rs = require("http/rs");
```

Comment:
Could you change this to an import statement?
```JS
import {rs} from '@dirigible/http'
```

Same for the other `require`s

Reply:
how to do it for `http`
in
```
const http = require("codbex-electra/gen/api/utils/http");
import { rs } from '@dirigible/http'
import { configurations } from '@dirigible/core'
import { logging } from '@dirigible/log'
```